src/components/molecules/Card/Card.js

```diff
@@ -2,76 +2,52 @@ import Title from "~/components/atoms/Title/Title";
 import Text from "~/components/atoms/Text/Text";
 import ImageAtomLocal from "~/components/atoms/imageAtom/ImageAtomLocal";
 
-export default function Card({
-  CardsData,
-  CardClassProps
-}) {
+export default function Card({ CardsData, CardClassProps }) {
+  // Calculate the number of cards in the last row
+  const remainder = CardsData?.length % 3;
+
   return (
-    <div className="flex flex-wrap mb-4 text-center items-center w-[100%] justify-center mt-5">
-      {CardsData?.map(({ route, name, text, subtitle, id }) => (    
-          <div 
-            className="flex flex-col text-center items-center card-content" // New class for the card content div
-            style={{ 
-              marginBottom: '30px',
-              marginTop: '4px',
-              borderRadius: '0px',
-              margin: '20px',
-              // height: '450px',
-              // boxShadow: '0px 0px 15px 0px rgba(0,0,0,0.75)',
-              // backgroundColor: 'rgba(0, 0, 0, 0.5)', // Add a semi-transparent background color to make text more readable
-              position: 'relative', // Add position relative to the card content div
-            }}
-            key={id}
-          >
-            <div 
-            style={{
-              position: 'absolute', top: 20, zIndex: 1 }}>
-            <Text
-              Style="subtitle"
-              Text={subtitle}
-              TextClassProps="text-white"
-            />
-              </div>
-              <div>
+    <div className="grid grid-cols-2 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-3 gap-4 mb-4 text-center items-center justify-center mt-5 w-full lg:w-[80rem] xl:w-[80rem] mx-[10rem] lg:mx-[5rem]">
+      {CardsData?.map(({ route, name, text, subtitle, id }, index) => (
+        <div 
+          className={`flex flex-col items-center space-y-4 p-4 w-full ${remainder === 1 && index === CardsData.length - 1 ? 'lg:col-start-2 xl:col-start-2' : ''}`}
```

Comment:
`lg:col-start-2` affects `xl` breakpoints. `xl:cols-start-2` shouldn't be needed.